src/app/(proper_react)/redesign/(public)/LandingView.tsx

```diff
@@ -79,7 +79,20 @@ export const View = (props: Props) => {
               {props.l10n.getString("landing-all-value-prop-fix-exposures")}
             </h2>
             <p>
-              <FixExposuresDescription />
+              {props.l10n.getFragment(
+                "landing-all-value-prop-fix-exposures-description",
+                {
+                  elems: {
+                    privacy_link: (
+                      <TelemetryLink
+                        eventData={{ button_id: "privacy_information" }}
+                        href="https://www.mozilla.org/en-US/firefox/privacy/"
```

Comment:
Nit: remove the /en-US/ path and let Bedrock handle best possible client languages.
```suggestion
                        href="https://www.mozilla.org/firefox/privacy/"
```